joomla/joomla-cms
# [com_content] Use condition show_tags in the view instead of in the model

#### Summary of Changes

Tags usually slow down content pages, so I changed code to not load tags when it has not been required.

Reasons:
- When show_tags is enabled in menu/global then modules which use "model articles" load unnecessary tags but did not use it. The PR removes unnecessary sql queries.
- Article view should not load tags if admin do not want to show tags.
- After applied patch there is option to show tags only for article view and hide for featured/categories. This is for those who love performance.
#### Testing Instructions

Change options for "Show tags" in menu items related to com_content.

1) Test menu item for featured view (test all options for "Show tags") and look at results.
2) Do the same for menu item of category view.
3) Check result on article views for above menu items.
